exporter/f5cloudexporter/README.md

```diff
@@ -0,0 +1,42 @@
+# F5 Cloud Exporter
+
+Exports data via HTTP to [F5 Cloud](https://portal.cloudservices.f5.com/).
+
+Supported pipeline types: metrics, traces, logs
+
+> :construction: This exporter is in beta and configuration fields are subject to change.
+
+## Getting Started
+
+The following settings are required:
+
+- `endpoint` (no default): The URL where to send data to. See your F5 Cloud account for details.
+- `source` (no default): A unique identifier that is used to distinguish where this data is coming from. This is in 
+  addition to the pipeline attributes and resources.
+- `f5_cloud_auth.credential_file` (no default): The credential file used to authenticate this client. See your F5 
```

Comment:
Since this exporter itself is `f5cloud` I think the namespace for auth settings can be just `auth`, not `f5_cloud_auth`

Reply:
I Was thinking that too; however, I noticed in the `awsprometheusremotewrite` exporter that they used 'aws_auth' ( 	AuthConfig AuthConfig `mapstructure:"aws_auth"` ), so I figured I would follow the same convention. The name duplication in the name bothers me, so if you believe it's unnecessary, I'll remove it.

Reply:
Yeah I'd say let's stick with `auth`